systemtests/src/main/java/io/enmasse/systemtest/platform/cluster/KubeClusterManager.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2016-2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.platform.cluster;
+
+import io.enmasse.systemtest.UserCredentials;
+import io.enmasse.systemtest.executor.Exec;
+import io.enmasse.systemtest.logs.CustomLogger;
+import io.enmasse.systemtest.platform.KubeCMDClient;
+import io.enmasse.systemtest.platform.Kubernetes;
+import io.enmasse.systemtest.time.TimeoutBudget;
+import io.enmasse.systemtest.utils.TestUtils;
+import io.fabric8.kubernetes.api.model.Node;
+import org.slf4j.Logger;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+/**
+ * Do the operation with cluster (scale, restart etc...)
+ */
+public class KubeClusterManager {
```

Comment:
Maybe rename OpenShiftClusterManager? I think almost everything here is openshift-specific.

Reply:
@lulf only worker scale is ocp specific, node drain is kubernetes stuff as well